include/drivers/uart.h

```diff
@@ -237,6 +239,55 @@ struct uart_event {
 	} data;
 };
 
+
+/**
+ * UART RS-485 transfer mode.
+ */
+typedef enum {
+	/** Full Duplex mode. */
+	UART_RS485_XFER_MODE_FULL_DUPLEX,
+
+	/** Software Controlled Half Duplex mode. */
+	UART_RS485_XFER_MODE_HALF_DUPLEX
+
+} uart_rs485_transfer_mode_t;
+
+/**
+ * UART RS-485 polarity for de and re signals.
+ */
+typedef enum {
+	/** Polarity for RS485 - Active Low  **/
+	UART_RS485_POL_ACTIVE_LOW = 0,
+
+
+	/** Polarity for RS485 - Active High  **/
+	UART_RS485_POL_ACTIVE_HIGH
+} uart_rs485_polarity_t;
+
+struct uart_rs_485_config {
```

Comment:
and here you call it `_rs_485_`. Can you please use one convention? (rs486 without underscore maybe)

Reply:
Thank for reviewing Anas,  will make the naming uniform.